README.md

```diff
@@ -1,8 +1,9 @@
+# Income Tax (Making Tax Digital) Changelog
 
-# income-tax-mtd-changelog
 
-The following Income Tax MTD services are supported:
+### List of services
 
+The changelog of the following Income Tax MTD services are supported:
```

Comment:
If we're keeping the list of services, could this be phrased something like:
"This page contains the latest changes for the following Income Tax MTD API services:"

Reply:
Updated as per feedback and pushed the changes